abodepy/__init__.py

```diff
@@ -137,6 +148,14 @@ def login(self, username=None, password=None):
 
         response_object = json.loads(response.text)
 
+        if response_object.get('mfa_type', None) == "google_authenticator":
```

Comment:
The only thing I might consider is adding an else to this block where if `mfa_type` exists in the response with any other value, return another exception of a type like `ERROR.UNKNOWN_MFA_REQUIRED`. That way if Abode ever does add another MFA type we can handle it gracefully.

Reply:
good call, ill add an error for unknown mfa type